### PR TITLE
ft(S3C-3020): Add functional tests for CreateCheckpoint

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

**Review by miniscruff — APPROVED** [review]

File: tests/functional/v2/task/testCreateCheckpoint.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+function assertCheckpoint(expected, checkpoint) {
+    assert.strictEqual(checkpoint.inB, expected.in);
+    assert.strictEqual(checkpoint.outB, expected.out);
+    assert.strictEqual(checkpoint.objD, expected.count);
+    assert.strictEqual(checkpoint.sizeD, expected.bytes);
+    assert.deepStrictEqual(checkpoint.ops, expected.ops);
+}
+
+
```
Comment (ilkescality) [inline tests/functional/v2/task/testCreateCheckpoint.js]: looks line an extra line

File: tests/functional/v2/task/testCreateCheckpoint.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    beforeEach(async () => {
+        prefix = uuid.v4();
+        checkpointTask = new CreateCheckpoint({ warp10: { nodeId: prefix } });
+        checkpointTask._program = { lag: 0, nodeId: prefix };
+
+        warp10 = new Warp10Client({ nodeId: prefix });
+    });
+
+
```
Comment (ilkescality) [inline tests/functional/v2/task/testCreateCheckpoint.js]: similarly

**Review by ilkescality — APPROVED** [review]

**Pull request merged**